Patches/Core/PawnKindDefs_Humanlike/PawnKinds.xml

```diff
@@ -744,8 +744,8 @@
     <value>
       <li Class="CombatExtended.LoadoutPropertiesExtension">
```

Comment:
Are these pawnkinds spawning strictly with pila, or are they sometimes spawning with bows? Cuz this isn't enough ammo for a combatant with a bow.